cart/cart.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>ShoppingCart</title>
+    <script type="module" src="shopping-cart.js"></script>
+</head>
+<body>
+    
+<table>
+    <thead>
+    <tr>
+        <th scope="col">Product Name</th>
+        <th scope="col">Price</th>
+        <th scope="col">Quantity</th>
+        <th scope="col">Subtotal</th>
+    </tr>
+    </thead>
+<tbody id="checkout-table">
+    <tr>
```

Comment:
You can delete the tr here, your function is generating them for you and you no longer need it.